techniques/system/server-common/1.0/reloadRudderServices.st

```diff
@@ -47,6 +49,8 @@ bundle agent system_reload_rudder_services {
     rudder_server_system_restart_jetty::
       "any" usebundle => service_restart("${jetty_service_name}");
     rudder_server_system_restart_apache::
+      "any" usebundle => service_restart("${apache_service_name}");
```

Comment:
with these changes, there never is any rudder_server_system_restart_apache defined - are you sure you haven't been too eager?

Reply:
It was never restarted, the "_restart_" class was only doing a reload.